config/tasklists/learn-to-drive-a-car.json

```diff
@@ -0,0 +1,175 @@
+{
+  "title": "Learn to drive a car: step by step",
+  "description": "Check what you need to do to learn to drive.",
+  "links": {
+    "breadcrumbs": [
+      { "title": "Home", "url": "/" },
+      { "title": "Driving and transport", "url": "/browse/driving"},
+      { "title": "Learning to drive", "url": "/browse/driving/learning-to-drive"}
+    ]
+  },
+  "tasklist": {
+    "steps": [
+      [
+        {
+          "title": "Check you're allowed to drive",
+          "panel": "Most people can start learning to drive when they’re 17.",
```

Comment:
You can use the 'panel' option here, but I'd recommend the 'panel_descriptions' as they're styled by the component, which means if we want to change how they look we can more easily do that. Panel descriptions are an array of strings, which get inserted into P tags, i.e.

panel_descriptions: [
          "This is a panel description.",
          "This is also a panel description. It has been filled with words such as CONTENT and ACCORDION in order to fill the horizontal space and wrap onto a second line."
        ]

Reply:
Ah, good shout, thanks.  I'll get these switched over and update the screenshots